misc/python/materialize/checks/debezium.py

```diff
@@ -122,6 +122,10 @@ def validate(self) -> Testdrive:
         return Testdrive(
             dedent(
                 """
+                # restart debezium after Postgres restart to avoid flakiness
```

Comment:
```
# Restart Debezium in case the Scenario in which this Check is being run restarts Postgres
# The Debezium connector is unable to handle all Postgres restarts -- it will stop replicating
# and require a manual restart via its REST API in order to continue.
```

Reply:
Thanks, updated